keps/sig-scheduling/3633-matchlabelkeys-to-podaffinity/README.md

```diff
@@ -758,13 +754,25 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+It shouldn't impact already running workloads. It's an opt-in feature,
```

Comment:
Can't comment on unchanged lines, so commenting here:

> Are there any tests for feature enablement/disablement?

Were those added?
In particular the comment from the template is relevant here.

Reply:
No... I'll add the one within this release, fortunately it's not too late like minDomains 😓

Reply:
@wojtek-t By the way, do we have to have a manual test (`Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?`) then? Is there any expectation difference between this automated enablement/disablement test and the manual test?

Reply:
The manual test should be performed as well. You can put the intent of the test in the KEP now and send an update later indicating that you executed it.